projects/plugins/jetpack/class.jetpack-plan.php

```diff
@@ -334,18 +344,6 @@ public static function supports( $feature ) {
 
 		$plan = self::get();
 
-		// Manually mapping WordPress.com features to Jetpack module slugs.
-		foreach ( $plan['features']['active'] as $wpcom_feature ) {
-			switch ( $wpcom_feature ) {
-				case 'wordads-jetpack':
```

Comment:
I think we no longer need this, since in D75985-code we're reporting `wordads` as the active feature (along with `wordads-jetpack` for old Jetpack sites).

Reply:
Just realized this won't work until https://github.com/Automattic/wp-calypso/issues/61542 is sorted our first, so I'm reverting this changes.